test/fuzz/common.proto

```diff
@@ -28,7 +28,7 @@ message ProtoBody {
   google.protobuf.Any message = 1 [(validate.rules).any.required = true];
 
   // The size (in bytes) of each buffer when forming the requests.
-  uint64 chunk_size = 2 [(validate.rules).uint64.gt = 0];
+  uint64 chunk_size = 2 [(validate.rules).uint64 = {gt: 0, lt: 8192}];
```

Comment:
@nareddyt Is this meant to be 8192? I think I'm seeing a regression.

Reply:
Right, i increased it to be more realistic. 20 was too small, and envoys buffer should handle this much data.

It was still working when running with libfuzzer for me. Could you share more details?

Reply:
https://github.com/envoyproxy/envoy/pull/11045 runs each fuzzer for a minute: https://circleci.com/gh/envoyproxy/envoy/347829?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
str_split.cc : 123] RAW: Check length > 0 failed: 
AddressSanitizer:DEADLYSIGNAL
=================================================================
==20698==ERROR: AddressSanitizer: ABRT on unknown address 0x0000000050da (pc 0x7f06f772be97 bp 0x7ffd0fe621b0 sp 0x7ffd0fe61070 T0)
    #0 0x7f06f772be96 in raise (/lib/x86_64-linux-gnu/libc.so.6+0x3ee96)
    #1 0x7f06f772d800 in abort (/lib/x86_64-linux-gnu/libc.so.6+0x40800)
    #2 0x469e2e23 in (anonymous namespace)::RawLogVA(absl::LogSeverity, char const*, int, char const*, __va_list_tag*) (/build/tmp/_bazel_bazel/b570b5ccd0454dc9af9f65ab1833764d/execroot/envoy/bazel-out/k8-fastbuild/bin/test/extensions/filters/http/common/fuzz/filter_fuzz_test_with_libfuzzer+0x469e2e23)
```

Maybe StrSplit should just not be used

Reply:
I think you can repro by making any of your corpus entries a 0 chunk size

Reply:
Thanks, I'll take a look and make a PR to fix

Reply:
I cannot reproduce this. I assume it's failing on this line here: https://chromium.googlesource.com/external/github.com/abseil/abseil-cpp/+/HEAD/absl/strings/str_split.cc#123

But the proto has validation to prevent this:
```
uint64 chunk_size = 2 [(validate.rules).uint64 = {gt: 0, lt: 8192}];
```

I confirmed setting this to 0 in the corpus results in validation failure:
```
[2020-05-21 20:01:05.178][12][debug][misc] [source/common/protobuf/utility.cc:255] Proto validation error; throwing Proto constraint validation failed (FilterFuzzTestCaseValidationError.Data: ["embedded message failed validation"] | caused by HttpDataValidationError.ProtoBody: ["embedded message failed validation"] | caused by ProtoBodyValidationError.ChunkSize: ["value must be inside range (" '\x00' ", " '\u2000' ")"]): config {
```

I also tested with:
- a empty `google.protobuf.Any`
- an Any with a type URL but no value

Neither crashes

Reply:
Ahhh! Thank you. I'm really sorry for causing trouble, it definitely can't be empty either with those validations. I think the reason might be because the PR *didn't* have your PR merged. I'm waiting for CI to pass after the merge, but thank you so much. I'm sorry about that!

Reply:
No problem! Good to double check edge cases :)